Assets/Scripts/SS3D/Systems/Interactions/RadialInteractionView.cs

```diff
@@ -39,14 +39,28 @@ public sealed class RadialInteractionView : Core.Behaviours.System
 
         private List<IInteraction> Interactions { get; set; }
         private InteractionEvent Event { get; set; }
+        private Controls.OtherActions _controls;
+
+        private void OnEnable()
+        {
+            try
```

Comment:
replace with` if(null && !didstart) return; else { Debug.Error(); return; }` So we can still have an error in case performed is null later in the game.